release-notes/v0.5.0.md

```diff
@@ -0,0 +1,95 @@
+# What's new in VS Code Java?
+
+*November 2018*
+
```

Comment:
Add a link to the web version somewhere in case someone wants to find the historical release notes?

Reply:
Future issues will have a section listing all the past issues.